cupy/cudnn.pyx

```diff
@@ -234,6 +235,16 @@ cpdef _create_convolution_descriptor(
         raise ValueError('groups must be one when cuDNN < 7.0')
 
 
+cpdef core.ndarray _ascontiguousarray_normalized_strides(core.ndarray a):
+    if a._c_contiguous:
```

Comment:
Explicitly type `newarray` for performance.
> cdef core.ndarray newarray

Reply:
I added `cdef` in `_ascontiguousarray_normalized_stride`.